code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -83,7 +83,17 @@
 	..()
 	if(panel_open)
 		to_chat(user, "<span class='notice'>[src]'s maintenance hatch is open!</span>")
-
+	if(in_range(user, src) || isobserver(user))
+		to_chat(user, "<span class='notice'>The status display reads: <br>Recharging <b>[recharge_amount]</b> power units per interval.<br>Power efficiency increased by <b>[(powerefficiency*1000)-100]%</b>.<span>")
+		switch(macrotier)
```

Comment:
This feels caveman tier, is there a more elegant way to do this? `macrotier` is 1->2->3->4 with part upgrades, but the actual granularity is 5u->3u->2u->1u.

Reply:
Please don't get me started on macro(megalul)tier

Reply:
why not just granularity at [5 - macrotier]?

Reply:
Because 6-macrotier results in 5-4-3-2 instead of 5-3-2-1.

Reply:
It can be 5-macrotier but you have to force macrotier at 1 similarly to how `get_resolution` or whatever does it.